recipes/trace-remote-sampling/README.md

```diff
@@ -0,0 +1,129 @@
+# Trace Remote Sampling Configuration Recipe
+
+This recipe shows how to use the Jaeger Remote Sampling
+protocol to provide fine-grained control of trace sampling.
+
+
+This recipe provides three main pieces of configuration:
+
+- A ConfigMap that provides fine-grained trace sampling controls for the entire cluster.
+- An OpenTelemetryCollector deployment that will serve the control protocol to instrumentation.
+- An Instrumentation configuration that will leverage the OpenTelemetryCollector deployment to look up trace sampling information.
+
+## Prerequisites
+
+* OpenTelemetry Operator installed in your cluster
+* Running un-instrumented application (such as one of the [sample apps](../../sample-apps)).
+
+## Running
+
+Apply the `ConfigMap` object from [`sampling-config.yaml`](sampling-config.yaml)
+
+```
+kubectl apply -f remote-sampling-config.yaml
+```
+
+This creates the configuration for trace sampling. At any point, we can modify the `remote-sampling-config.yaml` file and reperform this step to adjust sampling on the cluster in the future (see [Tuning](#tuning))
+
+
+Apply the `OpenTelemetryCollector` object from [`collector-config.yaml`](collector-config.yaml)
+
+```
+kubectl apply -f collector-config.yaml
+```
+
+Next, create the `Instrumentation` object from [`instrumetnation.yaml`](instrumentation.yaml) that will use the remote sampling service:
+
+```
+kubectl apply -f instrumentation.yaml
+```
+
+This creates a new object named `instrumentation/trace-remote-sampling` in the current namespace.
+
+Annotate your application pods to use these settings by editing the `instrumentation.opentelemetry.io`
+annotation using one of the following commands:
+
+> Note that if the app is a standalone Pod you can
+>`kubectl annotate` directly on the Pod, but if it is owned by a Deployment or other replica controller
+> you must patch the metadata of the Pod template.
+
+* **NodeJS:**
+
+  Pod:
+  ```
+  kubectl annotate pod/<APP-NAME> instrumentation.opentelemetry.io/inject-nodejs="trace-remote-sampling"
+  ```
+  Deployment:
+  ```
+  kubectl patch deployment.apps/<APP-NAME> -p '{"spec":{"template":{"metadata":{"annotations":{"instrumentation.opentelemetry.io/inject-nodejs": "trace-remote-sampling"}}}}}'
+  ```
+
+* **Java:**
+
+  Pod:
+  ```
+  kubectl annotate pod/<APP-NAME> instrumentation.opentelemetry.io/inject-java="trace-remote-sampling"
+  ```
+  Deployment:
+  ```
+  kubectl patch deployment.apps/<APP-NAME> -p '{"spec":{"template":{"metadata":{"annotations":{"instrumentation.opentelemetry.io/inject-java": "trace-remote-sampling"}}}}}'
+  ```
+
+* **Python:**
```

Comment:
I don't Python, JS (https://github.com/open-telemetry/opentelemetry-js/issues/692), or .NET have an implementation of Jaeger Remote sampler. Golang does https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/samplers/jaegerremote.

Reply:
I'll add a note and remove that for now.